CONTRIBUTING.md

```diff
@@ -166,18 +202,21 @@ Use "double quotes" wherever it's possible
             return;
         }
         ```
+
     - Correct
-        ```
+
+        ```javascript
         if (condition)
             console.log("Hello, World!");
         else
             return;
         ```
 
 ### Finding an issue to work on
+
 Check out the [full issues list](https://github.com/XFox111/TabsAsideExtension/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue) for a list of all potential areas for contributions. **Note** that just because an issue exists in the repository does not mean we will accept a contribution. There are several reasons we may not accept a pull request like:
 
-- Performance - One of Tabs Aside core values is to deliver a lightweight extension, that means it should perform well in both real and test environments.
+- Performance - One of the Tabs Aside core values is to deliver a lightweight extension, that means it should perform well in both real and test environments.
```

Comment:
I think the article "the" refers to the core values, not to Tabs Aside. Anyway, Grammarly says that is incorrect, so I corrected that

Reply:
Well, I've just consulted with my former English teacher. She says the article is unnecessary here.
Yeah, I used to use Grammarly. Quite cool tool tho. But not everything it suggests can be correct since sometimes it just misses a context